### PR TITLE
Feat/add optimize vector

### PR DESCRIPTION
**Pull request by AMoreaux**
## What it does

-  Optimize svg with svgo.

### CODE REVIEW

File: parsers/convert-font/README.md [hunk 1]
```diff
@@ -22,9 +20,6 @@ interface parser {
 | parameter | Require    | type      | default    | description                                       |
 | --------- | ---------- | --------- | ---------- | ------------------------------------------------- |
 | `formats`    | optional   | `Array<string>`   | `["woff2", "woff", "otf", "ttf", "eot"]` | the list of formats to convert |
```
Comment (ChucKN0risK) [inline parsers/convert-font/README.md:22]: I don't understand why the "optimize-vector" parser has font related parameters. Is this normal?

Comment (AMoreaux) [reply parsers/convert-font/README.md:22]: This is a mistake. Fixed.

File: parsers/global-libs.ts [hunk 2]
```diff
@@ -1,25 +1,29 @@
...
 import got from 'got';
-import { AllowedFormat } from '../types/tokens';
-import { PartialRecord } from '../types';
+import SVGO from 'svgo';
+import { AllowedFormat, PartialRecord } from '../types';
 
 const Libs = {
   _,
   tinycolor,
   got,
```
Comment (AllanMichay) [inline parsers/global-libs.ts:10]: is got still needed ?

Comment (AMoreaux) [reply parsers/global-libs.ts:10]: True. no.

**Pull request merged**